lib/python/qmk/path.py

```diff
@@ -65,3 +66,11 @@ def normpath(path):
         return path
 
     return Path(os.environ['ORIG_CWD']) / path
+
+
+class FileType(_FileType):
+    def __call__(self, string):
+        maybe_relative = Path(os.environ['ORIG_CWD'], string)
+        if maybe_relative.exists():
+            string = maybe_relative.resolve()
+        return super().__call__(string)
```

Comment:
```suggestion
        return super().__call__(normpath(string))
```

Reply:
Applying this suggestion would change the existing behaviour of allowing a relative file path to the qmk_firmware directory.

Reply:
I'd rather break advice given to work around a bug than continue to support that behavior. Users shouldn't care that internally `qmk` is operating from a different current directory.